lib/datadog/di/contrib/active_record.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Datadog::DI::Serializer.register(condition: lambda { |value| ActiveRecord::Base === value }) \
+do |serializer, value, name:, depth:|
+  value_to_serialize = {
+    attributes: value.attributes,
+  }
+  serializer.serialize_value(value_to_serialize, depth: depth ? depth - 1 : nil, type: value.class)
```

Comment:
Why the `- 1` in `depth - 1`?

Reply:
We went from the object to its attributes, this is one level in. I will double-check against other serialization code.

Reply:
We went from the object to its attributes, this is one level in. I will double-check against other serialization code.